course/src/main/assets/js_injection/survey_css.js

```diff
@@ -0,0 +1,27 @@
+var css = `
+    .survey-table:not(.poll-results) .survey-option label {
+        margin-bottom: 0px !important;
+    }
+
+    .survey-table:not(.poll-results) .survey-option .visible-mobile-only {
+        width: calc(100% - 21px) !important;
+    }
+
+    .survey-table:not(.poll-results) .survey-option input {
+        width: 13px !important;
+        height: 13px !important;
+    }
+
+    .survey-percentage .percentage {
+        width: 54px !important;
+    }`,
+head = document.head || document.getElementsByTagName('head')[0],
+style = document.createElement('style');
```

Comment:
We can declare these variables separately; to increase code readability. i.e. `var head` & `var style`.

Reply:
done